soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -809,7 +818,7 @@ void GenerateItemPool() {
   }
 
   if (BombchusInLogic) {
-    AddItemToMainPool(PROGRESSIVE_BOMBCHUS, 5);
+    AddItemToMainPool(PROGRESSIVE_BOMBCHUS, InfiniteUpgrades ? 2 : 5);//reduced with infinite upgrades because the second one gives infinite
```

Comment:
Others should weigh in here, but doing this changes the difficulty of the game, which may not be your intention.  Before this change, you'd have 5 separate chances to find bombchus, and with this change active, you'd only have 2.  From a logical standpoint, finding _any_ of those 5 would let you logically progress, so it stands to reason that you'd still want all 5 of those chances.

Reply:
These were copied from my original implementation from february. I agree. I removed this change.